Classes/vendor/MYUtilities/MYErrorUtils.m

```diff
@@ -139,22 +139,19 @@ BOOL MYMiscError( NSError **error, NSString *message, ... )
 #if !TARGET_OS_IPHONE || defined(__SEC_TYPES__)
     else if ($equal(domain,NSOSStatusErrorDomain)) {
         // If it's an OSStatus, check whether CarbonCore knows its name:
-        const char *name = NULL;
-#if !TARGET_OS_IPHONE
-        name = GetMacOSStatusErrorString((int)code);
-#endif
-        if (name && *name)
-            result = [NSString stringWithCString: name encoding: NSMacOSRomanStringEncoding];
-        else {
+        NSError *osErr = [NSError errorWithDomain:NSOSStatusErrorDomain code:code userInfo:nil];
+        result = [osErr localizedDescription];
```

Comment:
@indisoluble, excellent question. It appears that `GetMacOSStatusErrorString()` also returns a localized string, but I cannot tell you exactly what will happen.